packages/examples/packages/name-lookup/src/index.test.ts

```diff
@@ -57,7 +63,8 @@ describe('onNameLookup', () => {
       chainId: CHAIN_ID_MOCK,
     };
 
-    // @ts-expect-error - Testing invalid request.
+    const { onNameLookup } = await installSnap();
```

Comment:
We can remove this test case now since it is not possible to test due to validation. 

We validate the request here: https://github.com/MetaMask/snaps/blob/main/packages/snaps-execution-environments/src/common/commands.ts#L60

Reply:
Removed.